### PR TITLE
feat(workflow): Handle insights users with zero projects/teams

### PR DESCRIPTION
**Pull request by scttcper**
reuse the NoProjectMessage

adds no projects message to the existing components

image

### CODE REVIEW

File: static/app/views/teamInsights/teamMisery.tsx [hunk 1]
```diff
@@ -138,7 +140,8 @@ function TeamMiseryWrapper({
   const commonEventView = {
     id: undefined,
     query: 'transaction.duration:<15m team_key_transaction:true',
-    projects: projects.map(project => Number(project.id)),
+    // Empty array selects all projects for a user
+    projects: projects.length > 0 ? projects.map(project => Number(project.id)) : [-2],
```
Comment (matejminar) [inline static/app/views/teamInsights/teamMisery.tsx:144]: Why is there `[-2]`?

Comment (scttcper) [reply static/app/views/teamInsights/teamMisery.tsx:144]: -1 is all projects. Trying to get it to not query all projects. Maybe i should just return without making the queries at all

Comment (scttcper) [reply static/app/views/teamInsights/teamMisery.tsx]: returned early

**Comment by scttcper** [comment]
added a change that depends on #28988

**Review by matejminar — APPROVED** [review]

**Pull request merged**